### PR TITLE
Release

### PR DESCRIPTION
**Pull request by hluk**
Important bug fixes for Python 3.

### CODE REVIEW

File: bin/pdc_client [hunk 1]
```diff
@@ -151,6 +151,8 @@ if __name__ == "__main__":
                                         "debugging.",
                         action="store_true", default=False)
     parser.add_argument("-c", "--comment", help="Reasons for the PDC change.")
+    parser.add_argument('--version', action='version',
+                        version='%(prog)s ' + pdc_client.__version__)
```
Comment (xiangge) [inline bin/pdc_client]: This  pdc_client.__version__ is call the __version__ in __init__.py, right?
Maybe remove the line 22 as it has no use?

Comment (hluk) [reply bin/pdc_client]: Good point, but I'll just put `__version__` here instead.

Comment (xiangge) [reply bin/pdc_client]: Cool, that's also my idea!

**Comment by xiangge** [comment]
Looks good to me.

**Review by xiangge — APPROVED** [review]

**Review by simozhan — APPROVED** [review]
LGTM

**Comment by hluk** [comment]
This was merged to release branch (not master).

**Pull request closed without merging**